native/SampleApps/SmartSyncExplorerSwift/SmartSyncExplorerSwift/AppDelegate.swift

```diff
@@ -0,0 +1,268 @@
+ WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY
+ WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import UIKit
+import SalesforceSwiftSDK
+import MobileCoreServices
+import SalesforceSDKCore.SFPushNotificationManager
+import SalesforceSDKCore.SFDefaultUserManagementViewController
+import SalesforceSDKCore.SalesforceSDKManager
+import SalesforceSDKCore.SFUserAccountManager
+import SalesforceSDKCore.SFSDKAppConfig
+import SalesforceSDKCore.SFSDKWindowManager
+import SmartSync.SmartSyncSDKManager
+import SalesforceAnalytics.SFSDKDatasharingHelper
+import SalesforceAnalytics.NSUserDefaults_SFAdditions
+import SmartSyncExplorerCommon.SmartSyncExplorerConfig
+
+
+// Fill these in when creating a new Connected Application on Force.com
+//let RemoteAccessConsumerKey = "3MVG9Iu66FKeHhINkB1l7xt7kR8czFcCTUhgoA8Ol2Ltf1eYHOU4SqQRSEitYFDUpqRWcoQ2.dBv_a1Dyu5xa"
+//let OAuthRedirectURI        = "testsfdc:///mobilesdk/detect/oauth/done"
+
+let RemoteAccessConsumerKey = "3MVG9g9rbsTkKnAWkkUnzOrokwAGv9LB5W9tdPJEcQM9xMhNTQnnYHvCwjnjc3zQG5hQ_krRCb52wn0TU2kZH"
+let OAuthRedirectURI        = "com.salesforce.mobilesdk.sample.smartsyncexplorer://oauth2/success"
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+
+    override init() {
+        super.init()
+        
+        guard let config = SmartSyncExplorerConfig.sharedInstance() else { return }
+        SFSDKDatasharingHelper.sharedInstance().appGroupName = config.appGroupName
+        SFSDKDatasharingHelper.sharedInstance().appGroupEnabled = config.appGroupsEnabled
+        
+        SalesforceSwiftSDKManager.setInstanceClass(SmartSyncSDKManager.self)
+        SalesforceSDKManager.shared().appConfig?.remoteAccessConsumerKey = config.remoteAccessConsumerKey
+        SalesforceSDKManager.shared().appConfig?.oauthRedirectURI = config.oauthRedirectURI
+        SalesforceSDKManager.shared().appConfig?.oauthScopes = NSSet(array: config.oauthScopes) as! Set<AnyHashable>
+        
+        SalesforceSDKManager.shared().add(self)
+        
+        /*
+         // scheme of idpApp
+         SFUserAccountManager.sharedInstance().advancedAuthConfiguration = .none
+         SalesforceSwiftSDKManager.shared().idpAppURIScheme = "sampleidpapp"
+         // user friendly display name
+         SalesforceSwiftSDKManager.shared().appDisplayName = "RestAPIExplorerSwift"
+         
+         // Use the following code to replace the login flow selection dialog
+         SalesforceSwiftSDKManager.shared().idpLoginFlowSelectionBlock = {
+         let controller = IDPLoginNavViewController()
+         return controller as UIViewController & SFSDKLoginFlowSelectionView
+         }
+         */
+        
+        SalesforceSDKManager.shared().postLaunchAction = { [unowned self] (launchActionList: SFSDKLaunchAction) -> Void in
```

Comment:
It would be better to use `SalesforceSwiftSDKManager.init()` like you did [here](https://github.com/nem3/SalesforceMobileSDK-iOS/blob/20049654524dca03e78f23b97e2c0d62e143cbca/native/SampleApps/RestAPIExplorerSwift/RestAPIExplorerSwift/AppDelegate.swift#L67)

It would also take care of setting the correct instance class for the SDK manager.